<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1090</width>
    <height>125</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1090</width>
    <height>125</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="toolNoAndComment">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>5</y>
     <width>316</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Liberation Sans Narrow</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>T8 - Diam: 6 (comment for that tool)</string>
   </property>
  </widget>
  <widget class="QLabel" name="opNo">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>26</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Liberation Sans Narrow</family>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>##</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="opName">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>5</y>
     <width>120</width>
     <height>25</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>35</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Liberation Sans Narrow</family>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="text">
    <string>Face Milling</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="indent">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDeleteOp">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>5</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnEditOp">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>5</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </widget>
  <widget class="StatusLabel" name="spindleSpeed">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>55</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>35</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>25</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Liberation Sans Narrow</family>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>00000 CCW</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="rules" stdset="0">
    <string>[]</string>
   </property>
   <property name="format" stdset="0">
    <string>{:.0f}</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelOpName_13">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>55</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>50</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>35</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Liberation Sans Narrow</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="text">
    <string>Spindle RPM:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="indent">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="coolant">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>90</y>
     <width>56</width>
     <height>25</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>50</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>35</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Liberation Sans Narrow</family>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="text">
    <string>Flood</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="indent">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="labelOpName_16">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>76</width>
     <height>25</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>50</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>35</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Liberation Sans Narrow</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="text">
    <string>Coolant:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="indent">
    <number>0</number>
   </property>
  </widget>
  <widget class="StatusLabel" name="toZ">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>90</y>
     <width>60</width>
     <height>25</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>30</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>60</width>
     <height>25</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Liberation Sans Narrow</family>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>000.000</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="rules" stdset="0">
    <string>[]</string>
   </property>
   <property name="format" stdset="0">
    <string>{:.0f}</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelOpName_3">
   <property name="geometry">
    <rect>
     <x>335</x>
     <y>90</y>
     <width>36</width>
     <height>25</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>30</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>35</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Liberation Sans Narrow</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="text">
    <string>To Z:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="indent">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="labelOpName_2">
   <property name="geometry">
    <rect>
     <x>315</x>
     <y>55</y>
     <width>56</width>
     <height>25</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>50</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>35</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Liberation Sans Narrow</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="text">
    <string>From Z:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="indent">
    <number>0</number>
   </property>
  </widget>
  <widget class="StatusLabel" name="fromZ">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>55</y>
     <width>60</width>
     <height>25</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>30</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>60</width>
     <height>25</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Liberation Sans Narrow</family>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>000.000</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="rules" stdset="0">
    <string>[]</string>
   </property>
   <property name="format" stdset="0">
    <string>{:.0f}</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelOpName_7">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>90</y>
     <width>56</width>
     <height>25</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>50</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>35</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Liberation Sans Narrow</family>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="text">
    <string>mm/min</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="indent">
    <number>0</number>
   </property>
  </widget>
  <widget class="StatusLabel" name="stepOver">
   <property name="geometry">
    <rect>
     <x>565</x>
     <y>55</y>
     <width>60</width>
     <height>25</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>30</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>60</width>
     <height>25</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Liberation Sans Narrow</family>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>000.000</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="rules" stdset="0">
    <string>[]</string>
   </property>
   <property name="format" stdset="0">
    <string>{:.0f}</string>
   </property>
  </widget>
  <widget class="StatusLabel" name="xyFeed">
   <property name="geometry">
    <rect>
     <x>565</x>
     <y>90</y>
     <width>50</width>
     <height>25</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>30</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>25</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Liberation Sans Narrow</family>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>000.0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="rules" stdset="0">
    <string>[]</string>
   </property>
   <property name="format" stdset="0">
    <string>{:.0f}</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelOpName_4">
   <property name="geometry">
    <rect>
     <x>485</x>
     <y>55</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>50</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>35</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Liberation Sans Narrow</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="text">
    <string>Step Over:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="indent">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="labelOpName_6">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>90</y>
     <width>66</width>
     <height>25</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>50</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>35</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Liberation Sans Narrow</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="text">
    <string>XY Feed:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="indent">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="labelOpName_9">
   <property name="geometry">
    <rect>
     <x>985</x>
     <y>90</y>
     <width>76</width>
     <height>25</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>50</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>35</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Liberation Sans Narrow</family>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="text">
    <string>mm/flute</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="indent">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="labelOpName_15">
   <property name="geometry">
    <rect>
     <x>985</x>
     <y>55</y>
     <width>76</width>
     <height>25</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>50</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>35</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Liberation Sans Narrow</family>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="text">
    <string>mm/min</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="indent">
    <number>0</number>
   </property>
  </widget>
  <widget class="StatusLabel" name="chipLoad_2">
   <property name="geometry">
    <rect>
     <x>925</x>
     <y>55</y>
     <width>70</width>
     <height>25</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>35</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>70</width>
     <height>25</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Liberation Sans Narrow</family>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>000.000</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="rules" stdset="0">
    <string>[]</string>
   </property>
   <property name="format" stdset="0">
    <string>{:.0f}</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelOpName_14">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>55</y>
     <width>96</width>
     <height>25</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>50</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>35</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Liberation Sans Narrow</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="text">
    <string>Radial Speed:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="indent">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="labelOpName_8">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>90</y>
     <width>86</width>
     <height>25</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>50</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>35</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Liberation Sans Narrow</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="text">
    <string>Chip Load:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="indent">
    <number>0</number>
   </property>
  </widget>
  <widget class="StatusLabel" name="chipLoad">
   <property name="geometry">
    <rect>
     <x>925</x>
     <y>90</y>
     <width>70</width>
     <height>25</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>35</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>70</width>
     <height>25</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Liberation Sans Narrow</family>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>000.000</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="rules" stdset="0">
    <string>[]</string>
   </property>
   <property name="format" stdset="0">
    <string>{:.0f}</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>StatusLabel</class>
   <extends>QLabel</extends>
   <header>qtpyvcp.widgets.display_widgets.status_label</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
